flake.nix

```diff
@@ -0,0 +1,116 @@
+
+      let
+        pkgs = import nixpkgs {
+          inherit system;
+          overlays = [ qchem.overlay (import ./nix/overlay.nix) ];
+          config = {
+            allowUnfree = true;
+            qchem-config = {
+              optAVX = true;
+              allowEnv = true;
+            };
+          };
+        };
+      in {
+
+        legacyPackages = pkgs;
+
+        packages = {
+          pysisyphus = pkgs.python3.pkgs.pysisyphus;
+
+          pysisyphusLib = self.packages."${system}".pysisyphus;
+
+          pysisyphusApp = pkgs.pysisyphus;
+
+          pysisyphusOrca = pkgs.python3.pkgs.pysisyphus.override { enableOrca = true; };
+          pysisyphusTurbomole = pkgs.python3.pkgs.pysisyphus.override { enableTurbomole = true; };
+          pysisyphusFull = pkgs.python3.pkgs.pysisyphus.override {
+            enableOrca = true;
+            enableTurbomole = true;
+            enableCfour = true;
+            enableMolpro = true;
+            enableGamess = true;
+          };
+
+          pysisyphusDocker = pkgs.dockerTools.buildImage {
+            name = self.packages."${system}".pysisyphus.pname;
+            tag = self.packages."${system}".pysisyphus.version;
+
+            fromImageName = null;
+            fromImageTag = null;
+
+            contents = [
+              # Necessary for interactive usage
+              pkgs.bashInteractive
+              pkgs.coreutils
+
+              # Computational chemistry software
+              self.packages."${system}".pysisyphus
+            ];
+          };
+
+          pysisyphusSingularity =
+            pkgs.runCommand "${self.packages."${system}".pysisyphus.pname}.sif"
+            { } ''
+              cp -r /etc etc
+              touch etc/resolv.conf
+              ${pkgs.bubblewrap}/bin/bwrap \
+                --ro-bind /nix /nix \
+                --ro-bind etc /etc \
+                --ro-bind ${pkgs.bash}/bin /usr/local/bin \
+                --bind . /out \
+                --dev-bind /dev /dev \
+                --proc /proc \
+                --uid 1000 \
+                ${pkgs.singularity}/bin/singularity build /out/${
+                  self.packages."${system}".pysisyphus.pname
+                }.sif docker-archive://${
+                  self.packages."${system}".pysisyphusDocker
+                }
+              cp ${self.packages."${system}".pysisyphus.pname}.sif $out
+            '';
+        };
+
+        defaultPackage = self.packages."${system}".pysisyphusApp;
+
+        defaultApp = {
+          type = "app";
+          program = "${self.packages."${system}".pysisyphusApp}/bin/pysis";
+        };
+
+        devShell = let
+          pythonEnv = pkgs.python3.withPackages (p:
+            with p;
+            [ ] ++ p.pysisyphus.nativeBuildInputs ++ p.pysisyphus.buildInputs
+            ++ p.pysisyphus.propagatedBuildInputs);
+        in pkgs.mkShell {
+          buildInputs = [ pkgs.black pythonEnv ];
```

Comment:
pysis command is missing?!

Reply:
This is intended for how `nix develop` works. It gives you a shell to write and hack pysisyphus. Inside the shell you should be able to use standard python toolings and workflows. Those should then allow you to build Python with pip or setuptools or whatever.

If you want a shell, that has Pysisphus installed (which is then for using and not developing pysisyphus) you would do `nix shell`.